src/v/coproc/wasm_event.cc

```diff
@@ -158,9 +168,9 @@ validate_event(const model::record& r, parsed_event::event_header& header) {
 
 absl::btree_map<script_id, parsed_event>
 reconcile_events(std::vector<model::record_batch> events) {
-    absl::btree_map<script_id, parsed_event> wsas;
+    absl::btree_map<script_id, parsed_event> script_id_to_event;
     for (auto& record_batch : events) {
-        record_batch.for_each_record([&wsas](model::record r) {
+        record_batch.for_each_record([&script_id_to_event](model::record r) {
```

Comment:
can you split up this commit? I see a lot of stuff going on and its not really sure what is the commit is supposed to be doing. I see it adding tests, renaming variables, it looks like its removing unused code (event_listener::persist_actions ?), adds a file to cmakelists.txt with the file being in the commit, etc... each commit should do one atomic thing that can be described easily.

Reply:
Done
I tried that each commit can be build